infra/pr_helper.py

```diff
@@ -223,6 +228,26 @@ def get_pull_request_number(self, commit):
       return None
     return pr_response.json()[0]['number']
 
+  def get_past_contributors(self, project_path):
+    """Returns a list of past contributors of a certain project."""
+    commits_response = requests.get(f'{BASE_URL}/commits?path={project_path}',
+                                    headers=self._headers)
+
+    if commits_response.ok:
+      commits = commits_response.json()
+      contributors = set()
+      verified_contributors = []
+      for commit in commits:
+        login = commit['author']['login']
+        if login not in contributors:
+          verified = commit['commit']['verification']['verified']
```

Comment:
What if one commit is not verified, but another one is? 

I'd suggest doing something like this instead:

```python 
contributors : dict[str, bool] = {}  # Map of username to verified state (bool).

for commit in commits:
 ...
 if login not in contributors:
  contributors[login] = verified
 
 if verified:
  # Override previous verification bit. 
   contributors[login] = True
    
```

Reply:
Updated, sample comment: https://github.com/google/oss-fuzz/pull/10663#issuecomment-1628315454